modules/system/assets/ui/js/filter.js

```diff
@@ -302,6 +302,11 @@
 
         if (item)
             toItems.push(item)
+        else
+            toItems.push({
+                'id': itemId,
+                'name': $item.text()
+            })
```

Comment:
@danielbidala Please wrap conditions with curly brackets in JS:

```suggestion
        if (item) {
            toItems.push(item)
        } else {
            toItems.push({
                'id': itemId,
                'name': $item.text()
            })
        }
```

Reply:
@bennothommo I added a commit with curly brackets in conditions.